bench/tx-generator/tx-generator.cabal

```diff
@@ -98,7 +101,7 @@ library
                       , attoparsec-aeson
                       , base16-bytestring
                       , bytestring
-                      , cardano-api ^>= 8.38
+                      , cardano-api ^>= 8.38.0.0
```

Comment:
Out of curiosity, do those zeroes matter here?

Reply:
They don't think so. I am just using the version number that `cardano-api` uses.